package.json

```diff
@@ -48,7 +48,7 @@
 		"browserstack-local": "^1.4.8",
 		"chai": "^4.3.4",
 		"clean": "^4.0.2",
-		"commander": "^8.2.0",
+		"commander": "^8.3.0",
```

Comment:
Have you pushed the changes to package files on purpose?

Reply:
This change will be done in a separate PR.

Reply:
At my local it needs to update otherwise it was not getting further.

Reply:
The feel free to do so, but please do it in a separate PR. In other repos such things are also done separately from another.